packages/common/src/store/pages/feed/lineup/reducer.ts

```diff
@@ -11,15 +11,22 @@ export const initialState = {
   entryIds: new Set([])
 }
 
+type ResetSucceededAction = {
+  type: typeof RESET_SUCCEEDED
+}
+
 const actionsMap = {
-  [RESET_SUCCEEDED](state, action) {
+  [RESET_SUCCEEDED](_state: LineupState<Track>, _action: ResetSucceededAction) {
```

Comment:
feels dumb that we have to type these when we're not using them 😆

thoughts on this instead?
```
[RESET_SUCCEEDED]() {
  // ...
}
```